x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/configured_sources_list.tsx

```diff
@@ -112,7 +114,19 @@ export const ConfiguredSourcesList: React.FC<ConfiguredSourcesProps> = ({
                         hasMultipleConnectorOptions(sourceData) ? '' : 'connect'
                       }`}
                     >
-                      {CONFIGURED_SOURCES_CONNECT_BUTTON}
+                      {!connected
```

Comment:
Is `CONFIGURED_SOURCES_CONNECT_BUTTON` still imported?

Reply:
Gah, I keep forgetting to remove imports. Removed now, thanks.